src/main/kotlin/no/nav/syfo/Bootstrap.kt

```diff
@@ -140,7 +145,8 @@ fun main() {
     val smIkkeOkQueue = mqQueueManager.accessQueue(env.infotrygdSmIkkeOKQueue, openOptions)
 
     val accessTokenClient = AccessTokenClient(env.aadAccessTokenUrl, env.clientId, credentials.clientsecret)
-    val httpClient = HttpClient(Apache) {
+
```

Comment:
Hvordan fungerte den eksisterende accessTokenClient hvis vi ikke hadde en httpclient med proxy? Kan det være noe som bør ryddes opp i her?

Reply:
Hm, jeg er ikke sikker. Tenker du at det bør fungere uten proxcy for AccessTokenClientV2, eller at det er noe mystisk her?